weather_mv/loader_pipeline/sinks.py

```diff
@@ -380,6 +380,21 @@ def upload(src: str, dst: str) -> None:
     """Uploads a file to the specified GCS bucket destination."""
     subprocess.run(f'gsutil -m cp {src} {dst}'.split(), check=True, capture_output=True, text=True, input="n/n")
 
+def path_exists(path: str) -> bool:
```

Comment:
More cleaner logic: https://github.com/google/weather-tools/blob/main/weather_sp/splitter_pipeline/file_splitters.py#L85C6-L85C6.

Reply:
And for checking in local file system, we can have a local store like: https://github.com/google/weather-tools/blob/main/weather_dl/download_pipeline/stores.py#L80

Reply:
updated in 
427038e